mobile-app/app/screens/TransactionAuthorization/PasscodePrompt.tsx

```diff
@@ -36,148 +40,140 @@ interface PasscodePromptProps {
 }
 
 const PromptContent = React.memo((props: PasscodePromptProps): JSX.Element => {
+  const { isLight } = useThemeContext()
   return (
     <>
-      <ThemedTouchableOpacity
-        dark={tailwind('bg-gray-900')}
-        light={tailwind('bg-white')}
+      <ThemedTouchableOpacityV2
+        light={tailwind('bg-mono-light-v2-100')}
+        dark={tailwind('bg-mono-dark-v2-100')}
         onPress={() => props.onCancel(USER_CANCELED)}
-        style={tailwind('items-end pt-2 pr-2')}
+        style={[tailwind('items-end pt-6 pr-6'), { borderTopLeftRadius: 15, borderTopRightRadius: 15 }]}
         testID='cancel_authorization'
         disabled={[TransactionStatus.BLOCK, TransactionStatus.SIGNING].includes(props.status)}
       >
-        <ThemedIcon
-          dark={tailwind('text-white')}
-          light={tailwind('text-black')}
-          iconType='MaterialIcons'
-          name='close'
-          size={26}
-        />
-      </ThemedTouchableOpacity>
-
-      <ThemedView
-        dark={tailwind('bg-gray-900')}
-        light={tailwind('bg-white')}
+        <View
+          style={[tailwind(`w-6 h-6 flex justify-center items-center rounded-full ${isLight ? 'border-mono-light-v2-900' : 'border-mono-dark-v2-900'}`), { borderWidth: 1.5 }]}
+        >
+          <ThemedIcon
+            dark={tailwind('text-mono-dark-v2-900')}
+            light={tailwind('text-mono-light-v2-900')}
+            iconType='MaterialIcons'
+            name='close'
+            size={16}
+          />
+        </View>
+      </ThemedTouchableOpacityV2>
+      <ThemedViewV2
+        light={tailwind('bg-mono-light-v2-100')}
+        dark={tailwind('bg-mono-dark-v2-100')}
         style={tailwind('w-full flex-1 flex-col')}
       >
+        <ThemedViewV2
+          light={tailwind('bg-mono-light-v2-100')}
+          dark={tailwind('bg-mono-dark-v2-100')}
+        >
+          <ThemedTextV2
+            style={[tailwind('text-center font-normal-v2 px-16 pt-3.5'), { marginBottom: props.status !== TransactionStatus.SIGNING && props.status !== TransactionStatus.AUTHORIZED ? 76 : 0 }]}
```

Comment:
avoid inline style

Reply:
We need 76 margin for that.

Reply:
> 1. Passcode prompt heading is missing exact padding and margin as per figma
> 
> 
> <img alt="image" width="150" src="https://user-images.githubusercontent.com/53080940/177945131-4b134bff-6598-46b4-affb-4d8a8042750e.png">
> 
> <img alt="" width="150" src="https://user-images.githubusercontent.com/53080940/177945458-32dc1adf-72bd-4879-83cf-1d5ccdd52290.png">
> 2. there is flicker when user enter password and loader comes(position of input changes)
> RPReplay_Final1657270354.MP4
> 
> Also added few comments above

If we follow the figma exactly there will be an issue with the text unless we give the container a set width. In my opinion that's not ideal

Reply:
you can use combination of margin like mb-16 and on pin code add mt-3